### PR TITLE
fix(esl-share): fix focus behavior to a loop inside esl-share-popup

### PR DESCRIPTION
**Pull request by dshovchko**

### CODE REVIEW

**Review by ala-n — APPROVED** [review]

**Review by fshovchko — APPROVED** [review]

File: src/modules/esl-tooltip/core/esl-tooltip.ts [hunk 1]
```diff
@@ -43,21 +45,17 @@ export class ESLTooltip extends ESLPopup {
...
+  public get $focusables(): HTMLElement[] {
+    return getKeyboardFocusableElements(this) as HTMLElement[];
   }
 
-  /** Last focusable element inside Tooltip */
-  public get lastFocusableElement(): Element | null {
-    const els = this.focusableElements;
-    return els.length ? els[els.length - 1] : null;
+  /** First and last focusable elements inside instance */
+  public get $boundaryFocusable(): {$first: HTMLElement | undefined, $last: HTMLElement | undefined} {
```
Comment (NastaLeo) [inline src/modules/esl-tooltip/core/esl-tooltip.ts:54]: mb we can discuss to move it to helpers as the same focus loop is on the modal

Comment (NastaLeo) [reply src/modules/esl-tooltip/core/esl-tooltip.ts:54]: sorry for misunderstanding
previously it was discussed to leave this functionallity localy until focus trap will be implemented

**Review by NastaLeo — APPROVED** [review]

**Review by yadamskaya — APPROVED** [review]

**Review by abarmina — APPROVED** [review]

**Review by nattallius — APPROVED** [review]

**Pull request merged**